content/learn/fundamental-wellbeing.md

```diff
@@ -1,73 +1,96 @@
 ---
-title: "Fundamental Wellbeing"
-created: 2023-04-21
-authors: 
+title: An Introduction to Dr. Jeffery Martin's Work on Fundamental Wellbeing
+created: 2023-11-01
+authors:
   - rufuspollock
   - jefferymartin
+description: Fundamental Wellbeing is the the experience of persistent, higher states of conscious, also known as awakening, enlightenment and other terms. This page is an introduction to the concept.
+image: /assets/images/fundamental-wellbeing-dr-jeffery-martin.jpg
 ---
-## What is Fundamental Wellbeing
+ At Life Itself, we believe that cultivating being is one of the core tenants of creating a wiser, weller world. Greater spiritual enlightenment, leads to better wellbeing, greater compassion, and honed abilities to be connected to ourselves and others.
```

Comment:
I don't think we need the background here - that could come later - if at all (if we had such info i think it would likely be on a separate overview page where we point people to specific resources e.g. here's what to read on "waking up" aka FWB)